hare/preroundtracker.go

```diff
@@ -47,7 +47,7 @@ func (pre *preRoundTracker) OnPreRound(ctx context.Context, msg *Msg) {
 	logger := pre.logger.WithContext(ctx)
 
 	// check for winning VRF
-	vrfHash := hash.Sum(msg.Eligibility.Proof)
+	vrfHash := hash.Sum(msg.Eligibility.Proof.Bytes())
 	vrfHashVal := binary.LittleEndian.Uint32(vrfHash[:4])
```

Comment:
this is what i meant by inconsistent. we have 2 ways of comparing vrf values. 
not caused by your change of course.

Reply:
I updated the relevant code. Both in beacon and in hare the smallest VRF signature is now selected without hashing it first (but by comparing them litte instead of big endian) and for the coinflip the LSB of the signature is used.